install/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
```

Comment:
Take "INSTALL" and "DELETE" as arguments to your script, instead of 1 and 2.
Please change file name suitably.